src/erlzk_conn.erl

```diff
@@ -682,3 +684,6 @@ start_heartbeat(State = #state{timeout=Timeout}) ->
 
 def_pwd() ->
     <<0:128>>.
+
+rotate_server(#state{servers = [H|T]} = S) ->
+    S#state{servers = T ++ [H]}.
```

Comment:
This construct leads to quadratic complexity if there are frequent reconnects and many servers, but I'm assuming both of those conditions are false.

Reply:
Recommended number of servers is 5